lib/processors/background/BackgroundProcessor.ts

```diff
@@ -135,21 +133,28 @@ export abstract class BackgroundProcessor extends Processor {
       assetsPath += '/';
     }
 
-    this.maskBlurRadius = options.maskBlurRadius!;
     this._assetsPath = assetsPath;
-    this._debounce = typeof options.debounce === 'boolean' ? options.debounce : this._debounce;
-    this._debounceCount = this._debounce ? this._debounceCount : 1;
+    this._debounce = typeof options.debounce === 'boolean' ? options.debounce : false;
```

Comment:
Set debounce default to true, but emphasize setting it to false in the docs and example code.